cordova-plugin-outline/README.md

```diff
@@ -1,71 +1,31 @@
 # cordova-plugin-outline
 
-This Cordova plugin provides the ability to start a system-wide VPN, connected to a local Shadowsocks client.
+Cordova plugin that implements a system-wide VPN to proxy device traffic over [Shadowsocks](https://shadowsocks.org/).
 
-We use [tun2socks](https://github.com/ambrop72/badvpn-googlecode-export/blob/master/tun2socks/badvpn-tun2socks.8) as an adapter; it receives all of the device’s traffic through the VPN network interface (TUN) and forwards it to a local Shadowsocks server.
+## Supported Platforms
 
-## shadowsocks-libev
+This plugin supports Android, iOS, and macOS. Browser is supported for UI development/demo mode.
 
-### Android update instructions
-
-The client code is https://github.com/shadowsocks/shadowsocks-libev, is compiled to a native shared library using the same process as https://github.com/shadowsocks/shadowsocks-android.
-
-`libss-local.so` is built with the help of https://github.com/shadowsocks/shadowsocks-android.
-
-It should be refreshed periodically; a script is included to do so, e.g.:
-```bash
-./update-shadowsocks-libev.sh 2.6.3
-```
-
-Upon successful completion, the script stages a commit including the new binary.
+### Android
+This plugin targets Android devices running Lollipop (API 22), or higher.
 
-### Apple update instructions
+### Apple
+This plugin targets Apple devices running iOS 11.0+ and macOS/OS X 10.11.
 
-We use a custom framework dependency, Shadowsocks, that implements shadowsocks-libev. To update it see `../third_party/shadowsocks-libev/apple/README.md`.
 
 ## tun2socks
 
-### Android update instructions
-
-```
-ndk-build -C ../third_party/badvpn APP_BUILD_SCRIPT=Android.mk NDK_PROJECT_PATH=`pwd` NDK_APPLICATION_MK=`pwd`/../third_party/badvpn/Application.mk
-cp libs/armeabi-v7a/libtun2socks.so android/libs/armeabi-v7a/libtun2socks.so
-cp libs/x86/libtun2socks.so android/libs/x86/libtun2socks.so
-```
-
-For debug builds, which enable asserts and improved backtraces, append `NDK_DEBUG=1`.
-
-### Apple update instructions
-
-We use a custom framework dependency, PacketProcessor, that implements tun2socks. To update it see `../third_party/Potatso/README.md`
-
-## Target Platforms
-
-This plugin supports Android, iOS, and macOS.
+We rely on [outline-go-tun2socks](https://github.com/Jigsaw-Code/outline-go-tun2socks) to assemble IP traffic and implement the Shadowsocks protocol: it receives all of the device’s traffic through the VPN network interface (TUN) and forwards it to a [Shadowsocks proxy server](https://github.com/Jigsaw-Code/outline-ss-server).
 
-### Android
-This plugin targets Android devices running Lollipop (API 22), or higher.
+outline-go-tun2socks is consumed by native platforms as a library.
```

Comment:
This seems a little vague to me.  Which platforms are native?  And how do non-native platforms use it?

Reply:
I removed a reference to browser, which is not supported by the plugin.

> Which platforms are native?

The ones supported: Android, iOS, macOS.

> And how do non-native platforms use it?

They don't.